src/applications/edu-benefits/10203/tests/edu-10203.cypress.spec.js

```diff
@@ -0,0 +1,73 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: ['minimal-test', 'confirmation-stem-test'],
+    fixtures: { data: path.join(__dirname, 'schema') },
+
+    pageHooks: {
+      introduction: ({ afterHook }) => {
+        afterHook(() => {
+          cy.findAllByText(/start.+without signing in/i, { selector: 'button' })
+            .first()
+            .click();
+        });
+      },
+
+      'benefits/initial-confirm-eligibility': () => {
+        cy.findByText(/Continue/i, { selector: 'button' }).click();
+      },
+
+      'benefits/stem-eligibility': () => {
+        cy.get('@testKey').then(testKey => {
+          if (testKey === 'confirmation-stem-test') {
+            cy.get('input[id="root_isEnrolledStemNo"]').click();
+            // eslint-disable-next-line cypress/no-unnecessary-waiting
+            cy.wait(1500);
+            cy.get('input[id="root_isPursuingTeachingCertNo"]').click();
+            cy.get('input[id="root_benefitLeft_0"]').click();
+          } else {
+            cy.get('input[id="root_isEnrolledStemYes"]').click();
+            cy.get('input[id="root_benefitLeft_1"]').click();
+          }
+          cy.findByText(/Continue/i, { selector: 'button' }).click();
+        });
+      },
+
+      'benefits/confirm-eligibility': () => {
+        cy.get('@testKey').then(testKey => {
+          if (testKey === 'confirmation-stem-test') {
+            cy.get('.stem-eligibility-ul')
+              .find('fa-times')
+              .to.have.lengthOf(3);
+          }
+        });
+        cy.findByText(/Continue/i, { selector: 'button' }).click();
+      },
+    },
```

Comment:
where are the pages after `benefits/confirm-eligibility`?

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/14007#discussion_r476641431